hyperframe/frame.py

```diff
@@ -645,7 +645,7 @@ def parse_body(self, data):
 
         self.body_len = len(data)
         self.data = (
-            data[priority_data_length:len(data)-self.total_padding].tobytes()
+          data[priority_data_length:self.body_len-self.total_padding].tobytes()
```

Comment:
We can save the extra characters by changing `priority_data_length` to `priority_data_len`.